cookbooks/tomography_based_plate_motions/2D_slice_with_faults_slabs_and_topo.prm

```diff
@@ -0,0 +1,168 @@
+# This cookbook demonstrates how we can set up instantaneous mantle flow models
+# using available geophysical constraints. To avoid large computational cost,
+# this model is a 2D spherical shell with imposed plate boundaries and driven
+# by mantle forces arising from observed mantle heterogeneity.
+# The generated mantle flow field at the surface can be compared with the observed
+# surface deformation and the fit can be used to constrain the physical state
+# of the mantle.
+# We define plate boundaries and cratons using WorldBuilder.
+
+include 2D_slice_with_faults_and_cratons.prm
+
+set Additional shared libraries            = ./plugins/libtomography_based_plate_motions.so
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set Output directory                       = 2D-slice-with-faults-slabs-topo
+set World builder file                     = $ASPECT_SOURCE_DIR/cookbooks/tomography_based_plate_motions/input_data/world_builder_smac_cratons_faults_2D.json
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Start time                             = 0
+set End time                               = 0
+set Max nonlinear iterations               = 30
+set Adiabatic surface temperature          = 1573.0
+
+# We use matrix-free solver and geometric multigrid preconditioner
+# to reduce memory consumption.
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance                         = 1e-4
+    set Stokes solver type                              = block GMG
+    set Number of cheap Stokes solver steps             = 5000
+    set GMRES solver restart length                     = 500
+    set Maximum number of expensive Stokes solver steps = 0
+    set Use full A block as preconditioner              = true
+    set Linear solver A block tolerance                 = 1e-2
+  end
+
+  subsection Diffusion solver parameters
```

Comment:
As this is not a commonly used feature (at least to my knowledge), can you add some documentation above that notes this is the length scale that affects fields set to `prescribed field with diffusion`, which in this case is the slabs?